### PR TITLE
fix(Text): ensure text correctly toggles between dark and light modes…

### PR DESCRIPTION
**Pull request by AshNaz87**
… when viewing surahs

@naveed-ahmad `Day Mode` shows up in light mode
Screenshot 2020-04-02 at 21 19 13
Screenshot 2020-04-02 at 21 19 22

### CODE REVIEW

File: app/javascript/controllers/setting_controller.js [hunk 1]
```diff
@@ -165,12 +165,10 @@ export default class extends Controller {
...
         $("body").addClass("night");
+      } else if (isNightMode) {
+        $("body").addClass("night");
       } else {
-        if (isNightMode) {
-          $("body").addClass("night");
-        } else {
-          $("body").removeClass("night");
-        }
+        $("body").removeClass("night");
```
Comment (naveed-ahmad) [inline app/javascript/controllers/setting_controller.js:171]: if we really have to refactor this, how about following: 

```
if (e && e.matches || isNightMode) {
        $("body").addClass("night");
 } else{
        $("body").removeClass("night");
} 
```

File: app/javascript/controllers/setting_controller.js [hunk 2]
```diff
@@ -195,6 +193,9 @@ export default class extends Controller {
     const isNightMode = $("body").hasClass("night");
     $("body").toggleClass("night");
     $("#toggle-nightmode").toggleClass("text-primary");
+    var text = $("li a#toggle-nightmode span").text();
```
Comment (naveed-ahmad) [inline app/javascript/controllers/setting_controller.js:196]: We should not hard code localized messages. Use i18n, also let's make header consistent. Use the same grey color for Day mode/Night mode text.

File: app/javascript/controllers/setting_controller.js (hunk 1 above)
Comment (AshNaz87) [reply app/javascript/controllers/setting_controller.js:171]: Agreed

File: app/javascript/controllers/setting_controller.js (hunk 2 above)
Comment (AshNaz87) [reply app/javascript/controllers/setting_controller.js:196]: Should we use Nokogiri for this?

Couldn't find a "Rails" way of changing text based on the class of a HTML element

Comment (naveed-ahmad) [reply app/javascript/controllers/setting_controller.js:196]: @AshNaz87 check out how we've implemented the "copy ayah" feature. When text is copied, we're showing "Done" message. 

https://github.com/quran/quran.com-frontend-v2/blob/master/app/javascript/controllers/verse_controller.js#L64

Comment (naveed-ahmad) [reply app/javascript/controllers/setting_controller.js:196]: @AshNaz87 can you finish this today? We are going to have production build tonight iA

Comment (AshNaz87) [reply app/javascript/controllers/setting_controller.js:196]: Sure - when do you need this completed by exactly? My timezone is BST

**Pull request closed without merging**